package exception;

public class Throwexmple {

		public static void main(String[] args) 
		{ 
		// Use of unchecked Exception 
		/*try { 
			 double x=3/0; 
			throw new ArithmeticException(); 
		} 
		/*catch (ArithmeticException e) 
		{ 
			
			//System.out.println(e);
			e.printStackTrace(); 
		} 
			finally {
				System.out.println("hello");
			}*/

	throw new ArithmeticException(); 

		
		} 
		} 

	
	

